examples/scale_bar.py

```diff
@@ -21,7 +21,22 @@
     scale=(0.29, 0.26, 0.26),
 )
 viewer.scale_bar.visible = True
-viewer.scale_bar.unit = 'um'
+
+# Text options
+viewer.scale_bar.unit = 'um' # set to None to diplay no unit
+viewer.scale_bar.length = 23 # length, in units, of the scale bar
+viewer.scale_bar.font_size = 20 # default is 10
+
+# Text color
+viewer.scale_bar.colored = True # default value is False
+viewer.scale_bar.color = 'yellow' # default value is magenta: (1,0,1,1)
+
+# Background box
+viewer.scale_bar.box = True # add background box, default is False
+viewer.scale_bar.box_color = (1, 0, 1, 0.4) # this is magenta with alpha=0.4
```

Comment:
Can we use black here? I think almost every other color is distracting here. 😂 🙏 
```suggestion
viewer.scale_bar.box_color = (0, 0, 0, 0.4)  # black with alpha=0.4
```

Reply:
Of course, I intentionally chose a rough color so other people would give an opinion!

Reply:
Yeah, I obviously don't think anyone would use magenta for actual display, but I think that a strong color makes it more obvious what is happening, which is the point of the example.

Reply:
I think that users can match the black box color to the black box, and anything non-default is good here. In general, as a philosophy I think we should aim for our gallery examples to be pretty, which encourages folks to use the library.

(The case of the cropped text is a bit different, because it shows them that there's a bug, which is indeed important knowledge that you can't get without seeing it. But the box colour is pretty obvious either way?)

Reply:
I changed the color from magenta to cyan and lowered the alpha to 0.1 to it looks less egregious. I think a black box won't be very obvious given that most of the scale_bar resides outside the image in the example viewer. This way, a slight color can show the clipping issue and what the box is actually doing.